bin/configure-tower-projects.py

```diff
@@ -410,11 +409,47 @@ def set_participant_role(self, part_id: int, role: str) -> None:
         data = {"role": role}
         self.tower.request("PUT", endpoint, json=data)
 
+    def remove_participant(self, part_id: int) -> Dict:
+        """Remove a participant from a workspace
+
+        Args:
+            part_id (int): Participant ID for the user or team
+        """
+        endpoint = f"/orgs/{self.org.id}/workspaces/{self.id}/participants/{part_id}"
+        response = self.tower.request("DELETE", endpoint)
+        return response
+
+    def list_participants(self) -> Iterator:
+        """List the participants in a workspace"""
+        endpoint = f"/orgs/{self.org.id}/workspaces/{self.id}/participants"
+        participants = self.tower.paged_request("GET", endpoint)
+        return participants
+
+    def list_owner_participant_ids(self) -> List[int]:
+        """List the participant IDs of any workspace owners
+
+        This will include the workspace creator.
+        """
+        participants = self.list_participants()
+        owners = [part for part in participants if part["wspRole"] == "owner"]
+        owner_ids = [owner["participantId"] for owner in owners]
```

Comment:
Nit: I can't think of why it would be advantageous to have duplicated participant ids.  It could be beneficial returning a set here.

```
set(owner["participantId"] for owner in owners)
```

Reply:
Done!